tutorials/deploy-ubuntu-22-04-instance/index.mdx

```diff
@@ -9,12 +9,12 @@ tags: Ubuntu instance setup server
 categories:
   - compute
 dates:
-  validation: 2023-02-16
+  validation: 2023-08-22
   posted: 2022-08-11
 ---
 
 Ubuntu is one of the most popular Linux distributions.
-First released in 2004, Ubuntu, whose name is based on an ancient African word meaning ‘humanity to others', became quickly the favourite Linux distribution of users around the world, mostly because it is easy to install and use.
+First released in 2004, Ubuntu, whose name is based on an ancient African word meaning _humanity to others_, became quickly the favourite Linux distribution of users around the world, mostly because it is easy to install and use.
```

Comment:
```suggestion
First released in 2004, Ubuntu, whose name is based on an ancient African word meaning _humanity to others_, quickly became the favorite Linux distribution of users around the world, mostly because it is easy to install and use.
```